compiler/util/ir_data_utils.py

```diff
@@ -29,3 +29,11 @@ def to_json(self, *args, **kwargs):
   def from_json(data_cls: type[ir_data.Message], data):
     """Constructs an IR data class from the given JSON string"""
     return data_cls.from_json(data)
+
+
+def builder(target: ir_data.Message) -> ir_data.Message:
+  """Provides a wrapper for building up IR data classes.
+
+  This is a no-op and just used for annotation for now.
```

Comment:
Can the `for now` be more specific? Maybe just add a link to #118 .

Reply:
I have all the patches ready, ideally we'd just land them somewhat quickly and not bother but if it's going to take an extended amount of time we can update it.